fw/procfs.c

```diff
@@ -77,6 +86,18 @@ tfw_perfstat_collect(TfwPerfStat *stat)
 		SADD(serv.rx_bytes);
 		SADD(serv.tls_hs_successful);
 		SADD(serv.tls_hs_failed);
+
+		/*
+		 * Health statistics (differs from health monitor statistics):
+		 * total responses for tempesta
+		 */
+		if (stat->hm && pcp_stat->hm)
+			for (i = 0; i < stat->hm->ccnt; ++i) {
+				BUG_ON(stat->hm->rsums[i].code !=
```

Comment:
Is the `BUG_ON()` really needed here?

Reply:
I don't know, for me it looks more calming :)